### PR TITLE
Fix(Core/Script): Molten Core: Gehennas

### LINKED ISSUE #5458: [Molten Core] Gehennas
### [Molten Core] Gehennas

This issue is meant to track the issues with the Molten Core Boss "Gehennas"

## CURRENT BEHAVIOUR:

grafik

## EXPECTED BLIZZLIKE BEHAVIOUR:

- Correct timers

## SOURCE:

- Research done by @T1ti 

## STEPS TO REPRODUCE THE PROBLEM:

1. Fight Gehennas

## AC HASH/COMMIT:

d11d021f38e0e0db27e7c74da801402e7e5f17d9

## OPERATING SYSTEM:

- Universal

## MODULES:

- None

## CUSTOMIZATIONS:

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/98101612-molten-core-gehennas?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Hacki95**
Co-Author: @sanctum32 (Code)
Co-Author: @T1ti (Research)
Co-Author: @Hacki95 (Coordination)

## Changes Proposed:
- Adjust Gehennas Timers
- Adjust the timers for the spells from the adds
- Split Shadow Bolt into its two spells (One of the main aggro target and one for a random target)

## Issues Addressed:

- Closes #5458

## Tests Performed:

- Applied Patch
- Fought Gehennas

## How to Test the Changes:

1. Apply Patch
2. .tele moltencore
3. Fight Gehennas
4. Check the timers
5. To check Shadow Bolt you need to test it with several players.

## Known Issues and TODO List:

-/

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1620337536404777343.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1620337536404777343');
+
+-- Timers update for NPC "Flamewaker Protector"
+UPDATE `smart_scripts` SET `event_param1`='5000', `event_param2`='5000', `event_param3`='6500', `event_param4`='6500' WHERE `entryorguid`=12119 AND `source_type`=0 AND `id`=0 AND `link`=0;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1620337536404777343.sql:4]: dont use '' around numbers unless they are strings

**Comment by claudiodfc** [comment]
How does this PR fix `Molten core hounds`?

**Comment by claudiodfc** [comment]
## Tested this on Windows 10.
### Commands
.cheat god
.gm on
.go c id 12259
.gm off

### **Gehennas** spells during encounter
Result | Spell | Time window
--- | --- | ---
✔️ | Gehennas Curse | 6 - 9 seconds
✔️ | Gehennas Curse repeat | 25 - 30 seconds
✔️ | Shadow Bolt | 3 to 5 seconds
✔️ | Shadow Bolt repeat | 5 seconds
✔️ | Rain of Fire repeat | 6 seconds

**Comment by Hacki95** [comment]
> 
> 
> How does this PR fix `Molten core hounds`?

Uhm you are right, wrong PR. Thanks for your tests! Edited.

**Comment by claudiodfc** [comment]
@FrancescoBorzi Ready to be merged

**Comment by FrancescoBorzi** [comment]
good job @Hacki95 @claudiodfc @T1ti @sanctum32 , nice team work

**Review by FrancescoBorzi — APPROVED** [review]

**Pull request merged**